client/components/auth-form-login.jsx

```diff
@@ -18,6 +19,26 @@ export default class AuthFormLogin extends React.Component {
     this.setState({ [name]: value });
   }
 
+  handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('/api/auth/sign-in', req)
+      .then(res => res.json())
+      .then(result => {
+        if (result.user && result.token) {
+          this.props.onSignIn(result);
+        }
+      });
+    this.setState({ isAuthorized: true });
+    this.props.onCloseAuthModal();
```

Comment:
These two lines aren't quite right.

`isAuthorized` isn't really useful. You should base this fact on whether or not your App has a user object in its state.

with regards to calling `this.props.onCloseAuthModal()`, that shouldn't be called unless you know that the user is done logging in, so it should go in your conditional code block that confirms that the response includes the user and token